File: android/src/main/java/expo/modules/libvlcplayer/records/Tracks.kt
package expo.modules.libvlcplayer.records

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record
import java.io.Serializable

class Tracks(
    @Field var audio: Int = 0,
    @Field var video: Int = 0,
    @Field var subtitle: Int = 0,
) : Record,
    Serializable

File: ios/Records/Tracks.swift
import ExpoModulesCore

struct Tracks: Record, Equatable {
    @Field
    var audio: Int = 0

    @Field
    var video: Int = 0

    @Field
    var subtitle: Int = 0

    static func == (lhs: Tracks, rhs: Tracks) -> Bool {
        lhs.audio == rhs.audio &&
            lhs.video == rhs.video &&
            lhs.subtitle == rhs.subtitle
    }
}
